main.go

```diff
@@ -114,18 +116,6 @@ func (s *DB) New() *DB {
 	return clone
 }
 
-type closer interface {
-	Close() error
-}
-
-// Close close current db connection.  If database connection is not an io.Closer, returns an error.
-func (s *DB) Close() error {
```

Comment:
This has been removed and the dude will never put it back: https://github.com/go-gorm/gorm/issues/5238#issuecomment-1101913873